core/src/test/scala/org/apache/spark/PreemptionPolicySuite.scala

```diff
@@ -0,0 +1,147 @@
+/*
```

Comment:
Is it possible to have a stress test (i.e. process a large number of randomized executor creation and preemption messages and make sure nothing crashes) that is not enabled by default but is still checked in here?